code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -176,4 +177,4 @@
 	new /obj/item/reagent_containers/dropper(src)
 	new /obj/item/storage/box/syringes/variety(src)
 	new /obj/item/storage/box/beakers/variety(src)
-	new /obj/item/clothing/glasses/science(src)
\ No newline at end of file
+	new /obj/item/clothing/glasses/science(src)
```

Comment:
Yeah I really dont understand what happened here, it probably just added a extra blank space at the end?

Reply:
your program added a newline at the end of the file automatically.

This is a good thing and you should not worry about it